infra/terraform/test-org/org/locals.tf

```diff
@@ -973,5 +973,18 @@ locals {
         ENABLE_BPMETADATA = "1"
       }
     },
+    {
+      name        = "terraform-google-enterprise-genai"
+      short_name  = "enterprise-genai"
```

Comment:
I don't think we need `short-name` as this repo follows the standard `terraform-google-*` convention.

Reply:
removed